<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>FormSelectMap</class>
 <widget class="QDialog" name="FormSelectMap" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Select map</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" colspan="2" >
    <widget class="QFrame" name="frameMaps" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>3</hsizetype>
       <vsizetype>3</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QListWidget" name="listMaps" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>1</hsizetype>
       <vsizetype>7</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="2" column="1" >
      <widget class="QLabel" name="labelNodeCount" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item rowspan="3" row="5" column="0" colspan="2" >
      <widget class="QTextBrowser" name="textDescription" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Nodes:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2" >
      <widget class="QLabel" name="labelName" >
       <property name="font" >
        <font>
         <family>Sans Serif</family>
         <pointsize>9</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text" >
        <string>TextLabel</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Author:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLabel" name="labelAuthor" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
       <property name="textFormat" >
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Playerlimit:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <widget class="QLabel" name="labelSpawnPoints" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>FormSelectMap</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>278</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>FormSelectMap</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>369</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
